ds3-metadata/src/main/java/com/spectralogic/ds3client/metadata/WindowsMetadataStore.java

```diff
@@ -168,7 +168,14 @@ private void saveWindowsfilePermissions(final Path file) throws IOException {
             final StringBuilder userDisplayList = new StringBuilder();
             final Map<String, Set<Integer>> stringSetMap = new HashMap<>();
             for (final AclEntry aclEntry : aclEntries) {
-                userDisplay = aclEntry.principal().getName().split("\\\\")[1];
+                final String[] principalFields = aclEntry.principal().getName().split("\\\\");
+
+                if (principalFields.length < 2) {
```

Comment:
A comment here explaining this behavior would be useful to refer to in the future